LongWarOfTheChosen/Src/LW_Overhaul/Classes/UIOutpostManagement.uc

```diff
@@ -83,176 +94,347 @@ function OnRadioTowerUpgradeClicked(UIButton Button)
-    NameHeaderButton.SetPosition(0, 0);
+	NameHeaderButton.bAnimateOnInit = false;
+	NameHeaderButton.bIsNavigable = false;
+	NameHeaderButton.ResizeToText = false;
+    NameHeaderButton.InitButton(, CAPS(m_strName));
+	NameHeaderButton.SetPosition(0, 0);
+    NameHeaderButton.SetSize(AvailableSpace * m_NameHeaderPct, 30);
+	NameHeaderButton.SetStyle(eUIButtonStyle_NONE, m_HeaderFontSize);
     NameHeaderButton.SetWarning(true);
+	// KDM : Since the name column header can't be clicked, remove its hit testing so mouse events don't change its colour
+	// and make users think the button is active. The same is done for the perks column header below.
+	NameHeaderButton.SetHitTestDisabled(true);
+
+	NextX = NameHeaderButton.X + NameHeaderButton.Width + 2;
 
+	// KDM : Rebel perks column header used in the fancy UI
+	if (m_bUseFancyVersion)
+	{
+		PerksHeaderButton = Spawn(class'UIButton', HeaderPanel);
+		PerksHeaderButton.bAnimateOnInit = false;
+		PerksHeaderButton.bIsNavigable = false;
+		PerksHeaderButton.bProcessesMouseEvents = false;
+		PerksHeaderButton.ResizeToText = false;
+		PerksHeaderButton.InitButton(, m_strPerks);
+		PerksHeaderButton.SetPosition(NextX, 0);
+		PerksHeaderButton.SetSize(AvailableSpace * m_PerksHeaderPct, 30);
+		PerksHeaderButton.SetStyle(eUIButtonStyle_NONE, m_HeaderFontSize);
+		PerksHeaderButton.SetWarning(true);
+		PerksHeaderButton.SetHitTestDisabled(true);
+	
+		NextX += PerksHeaderButton.Width + 2;	
+	}
+
+	// KDM : Rebel job column header
     JobHeaderButton = Spawn(class'UIButton', HeaderPanel);
-    JobHeaderButton.InitButton(, m_strMission, OnJobHeaderButtonClicked);
-    JobHeaderButton.SetResizeToText(false);
-    JobHeaderButton.SetWidth(HeaderPanel.Width * 0.3f);
-    JobHeaderButton.SetPosition(NameHeaderButton.X + NameHeaderButton.Width + 2, 0);
+	JobHeaderButton.bAnimateOnInit = false;
+	JobHeaderButton.bIsNavigable = false;
+	JobHeaderButton.ResizeToText = false;
+    JobHeaderButton.InitButton(, CAPS(m_strMission), OnJobHeaderButtonClicked);
+    JobHeaderButton.SetPosition(NextX, 0);
+	JobHeaderButton.SetSize(AvailableSpace * m_JobHeaderPct, 30);
+	JobHeaderButton.SetStyle(eUIButtonStyle_NONE, m_HeaderFontSize);
     JobHeaderButton.SetWarning(true);
+	JobHeaderButton.ProcessMouseEvents(OnJobHeaderMouseEvent);
 
     NextY += 32 + 5;
- 
+
+	// KDM : List container which will hold rows of rebels
 	List = Spawn(class'UIList', MainPanel);
+	List.bAnimateOnInit = false;
 	List.bIsNavigable = true;
-	List.InitList(, 15, NextY, ListBG.Width - 50, panelH - NextY - 5);
-	List.bStickyHighlight = false; 
+	List.bStickyHighlight = false;
+	List.InitList(, m_BorderPadding, NextY, HeaderPanel.Width, panelH - NextY - m_BorderPadding);
 
-	//check to see if we are in geoscape
-	if (`HQGAME == none || `HQPRES == none || `HQPRES.StrategyMap2D == none)
+	// KDM : As per Long War 2, radio relays can only be built via the strategy map, not via the Avenger.
+	if (!class'Utilities_LW'.static.IsOnStrategyMap())
```

Comment:
Can't we just make this a single `if` rather than an `if-else` with an empty `if` branch? I know it's not your code, but it makes not sense to have an empty branch.